tutorials/cleanrl.py

```diff
@@ -1,3 +1,15 @@
+"""Basic code which shows what it's like to run PPO on the Pistonball env using the parallel API, this code is inspired by CleanRL.
+
+This code is exceedingly basic, with no logging or weights saving.
+The intention was for users to have a (relatively clean) ~200 line file to refer to when they want to design their own learning algorithm.
+
+Dependencies:
+- SuperSuit==3.6.0
+- numpy==1.23.2
```

Comment:
numpy being stated not needed. If concerned about numpy version control, move it to the last dependency. imo I'd remove it